webpack.config.js

```diff
@@ -134,5 +147,7 @@ module.exports = {
 				},
 			},
 		] ),
+		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
```

Comment:
There are two things that are removed by this change. I believe the webpack bundle analyzer and the live reload config. I'm not sure if people find these useful or not but I thought I'd mention.

Reply:
> There are two things that are removed by this change. I believe the webpack bundle analyzer and the live reload config. I'm not sure if people find these useful or not but I thought I'd mention.

Oof. Well, I can go conservative and leave the behavior unaffected, with the downsides of (a) more duplication and (b) more changes to `package.json` and `package-lock.json` or (b) just drop them. I've personally not made use of them (though in the former case, I _have_ done bundle analysis by just calling `webpack --stats` directly).

Not sure yet how I want to proceed.

Reply:
> I'm personally fine with just removing those as not much used but it could be done separately and I think we can move forward here.

I'll commit to reintroducing it if anyone mentions it.